tests/integration/test_wazuh_db/test_db_backup.py

```diff
@@ -0,0 +1,233 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh-db is the daemon in charge of the databases with all the Wazuh persistent information, exposing a socket
+       to receive requests and provide information. The Wazuh core uses list-based databases to store information
+       related to agent keys, and FIM/Rootcheck event data.
+       This test checks the usage of the backup command used generate backups and restore the database using backups
+       generated with this same command.
+
+tier: 0
+
+modules:
+    - wazuh_db
+
+components:
+    - manager
+
+daemons:
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+
+tags:
+    - wazuh_db
+'''
+
+import os
+import time
+
+import pytest
+import yaml
+
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.modules import TIER0
+from wazuh_testing.wazuh_db import query_wdb
+from wazuh_testing.tools.file import recursive_directory_creation, remove_file
+
+
+# Marks
+pytestmark = [pytest.mark.linux, TIER0, pytest.mark.server]
```

Comment:
Could you use the variabled defined for linux, and server, please? The variables are on `modules/__init__.py`.

Reply:
fixed